### PR TITLE
[Develop] Mauricio - Environment reader

### LINKED ISSUE #2: Base Project - Environment Reader
Acceptance Criteria:
- Create a config.json.dist file that should contain different accounts with their corresponding API key and token properties.
- Create utility methods to access the environment data.
- Use gradle.properties
- Add 'threadCount'

**EP**(1-10) = 6

### PR DESCRIPTION
**Pull request by Snip-77**

### CODE REVIEW

File: src/main/java/org/fundacionjala/trello/config/Environment.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    private static final String VERSION = "version";
+    private static final String THREAD_COUNT = "threadCount";
+    private static final String PROPERTIES_FILE_PATH = "gradle.properties";
+    private Properties properties;
+
+    /**
+     * Configures the environment properties from gradle.properties.
+     * Initializes instance of PropertiesUtils class.
+     */
+
```
Comment (luqp) [inline src/main/java/org/fundacionjala/trello/config/Environment.java]: Here there is a unnecessary space.

File: gradle.properties [hunk 2]
```diff
@@ -1 +1,6 @@
 filterTags=
+baseUri=https://api.trello.com
+apiKey=
+apiToken=
+version=/1
```
Comment (luqp) [inline gradle.properties]: I think the **version** keyword should only contain numbers, and you can put "/" in the **baseUri** keyword:
`baseUri=https://api.trello.com/`

**Review by luqp — COMMENTED** [review]
Hello, good job, and I found some changes to improve the code, please take a look.

File: gradle.properties (hunk 2 above)
Comment (Snip-77) [reply gradle.properties]: Good observation, working on it....

**Review by marcos-x86 — APPROVED** [review]

**Review by marcos-x86 — APPROVED** [review]
LGTM!

**Pull request merged**